tests/phpunit/tests/Story_Post_Type.php

```diff
@@ -231,13 +221,4 @@ public function test_show_admin_bar() {
 		$show_admin_bar   = $post_type_object->show_admin_bar( 'current' );
 		$this->assertFalse( $show_admin_bar );
 	}
-
-	/**
-	 * @covers ::replace_editor
-	 */
-	public function test_replace_editor() {
```

Comment:
Why is being removed?

Reply:
It's explained in PR description. Can't really test that method properly because it loads a template which itself does a lot of things including loading `wp-admin/admin-header.php` etc.